quirks/password-rules.json

```diff
@@ -102,6 +102,10 @@
     "girlscouts.org": {
         "password-rules": "minlength: 8; maxlength: 16; required: lower; required: upper; required: digit; allowed: [abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789$#!];"
     },
+    "gov.br": {
+        "password-rules": "minlength: 8; maxlength: 70; required: [!"#$%&()*+,-./:;<=>?@[\]^{|}~.];"
+        "exact-domain-match-only": true
```

Comment:
So you’re trying to say that no subdomains of gov.br should have these rules?

Reply:
edited to match actua rules.